hls-writer/hls_writer.py

```diff
@@ -35,8 +36,14 @@ def hls_writer(layer_list, yamlConfig):
         elif '//hls-fpga-machine-learning insert weights' in line:
             newline = line
             for i in range(1,len(layer_list)+1):
-                newline += '#include "weights/w{}.h"\n'.format(i)
-                newline += '#include "weights/b{}.h"\n'.format(i)
+                if layer_list[i-1]['class_name'] != 'BatchNormalization' and layer_list[i-1]['class_name'] != 'Activation':
```

Comment:
I see Activation several places below.  What is this for?  I think we haven't come across this before.